src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/jclouds/openstack/OpenstackJCloudsProvider.java

```diff
@@ -209,4 +221,14 @@ private final Instance createInstanceFromNode(Server server) {
                        .build();
     }
 
+    @Override
+    public RunScriptOptions getRunScriptOptionsWithCredentials(InstanceCredentials credentials) {
+        logger.info("Script options: username=" + credentials.getUsername() + ", password=(given in credentials)");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this hard-coded password. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS2068)